integration-tests/src/slowIntegrationTest/java/pl/allegro/tech/hermes/integrationtests/AttachingKeepAliveHeaderTest.java

```diff
@@ -21,23 +19,15 @@
 
 public class AttachingKeepAliveHeaderTest {
 
+    @Order(0)
```

Comment:
Are we sure we need to use this annotation? I thought the tool was smart enough to take care of the order itself

Reply:
I'm not sure, but to me, the contract of `@RegisterExtension` seems too vague to assume that `@Order` is unnecessary in this case: https://junit.org/junit5/docs/5.4.0/api/org/junit/jupiter/api/extension/RegisterExtension.html